dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackerdataview/TrackerDataView.java

```diff
@@ -60,10 +60,10 @@ public class TrackerDataView implements Serializable
     @JsonProperty
     @JacksonXmlProperty( namespace = DxfNamespaces.DXF_2_0 )
     @Builder.Default
-    private Set<String> attributes = new HashSet<>();
+    private Set<String> attributes = new LinkedHashSet<>();
```

Comment:
You think you could explain the reason for not using a List? Uniqueness?

Reply:
I think it will be useful add a test in `web-api-test` that goes through the different possible cases (uniqueness, order...)

Reply:
Remember that if the field is still a `Set` the setter can be called with any implementation including those that do not care about order. This might be a problem. Can't say without knowing the case here.